spec/feature/unauthenticated_guest_spec.rb

```diff
@@ -130,25 +128,25 @@
                 id: 1,
                 display_name: "joe")
     Listing.create(title: "Rainy day house",
-                  description: "house in the rain",
-                  category_id: 1,
-                  max_guests: 4,
-                  nightly_rate: 20000,
-                  address1: "124 Elm St",
-                  address2: nil,
-                  city: "Seattle",
-                  state: "WA",
-                  zip: 98106,
-                  shared_bathroom: true,
-                  user_id: 1)
+                   description: "house in the rain",
+                   category_id: 1,
+                   max_guests: 4,
+                   nightly_rate: 20000,
+                   address1: "124 Elm St",
+                   address2: nil,
+                   city: "Seattle",
+                   state: "WA",
+                   zip: 98106,
+                   shared_bathroom: true,
+                   user_id: 1)
     visit listings_path
-    find(:css, "#category_id_1[value='1']").set(true)
-    click_link_or_button "Filter by listing category"
+    select('House', :from => 'category_id')
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.